src/main/java/com/google/api/generator/spring/composer/SpringAutoConfigClassComposer.java

```diff
@@ -955,6 +1061,7 @@ private static Map<String, TypeNode> createDynamicTypes(Service service, String
     typeMap.put("ServiceClient", serviceClient);
     typeMap.put("ServiceSettings", serviceSettings);
     typeMap.put("ServiceSettingsBuilder", serviceSettingsBuilder);
+    typeMap.put("GlobalProperties", GlobalPropertiesUtils.getGlobalPropertiesType());
```

Comment:
We will need to add a concrete reference once we merge https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1308

Reply:
Solved as we have full control of the global properties in spring-cloud-gcp